pkg/controller.v1beta1/suggestion/suggestion_controller.go

```diff
@@ -241,7 +241,7 @@ func (r *ReconcileSuggestion) ReconcileSuggestion(instance *suggestionsv1beta1.S
 	if foundDeploy, err := r.reconcileDeployment(deploy, suggestionNsName); err != nil {
 		return err
 	} else {
-		if isReady := r.checkDeploymentReady(foundDeploy); isReady != true {
+		if isReady := r.checkDeploymentReady(foundDeploy); !isReady {
```

Comment:
Should we simplify it ?
```suggestion
		if !r.checkDeploymentReady(foundDeploy) {
```

Reply:
It makes sense!